[2022-05-07 07:01:43,533] {processor.py:163} INFO - Started process (PID=8747) to work on /opt/airflow/dags/padron.py
[2022-05-07 07:01:43,537] {processor.py:642} INFO - Processing file /opt/airflow/dags/padron.py for tasks to queue
[2022-05-07 07:01:43,539] {logging_mixin.py:109} INFO - [2022-05-07 07:01:43,538] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/padron.py
[2022-05-07 07:01:43,612] {logging_mixin.py:109} INFO - [2022-05-07 07:01:43,598] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/padron.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/padron.py", line 4, in <module>
    from airflow.providers.apache.hive.operators.hive import HiveOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-05-07 07:01:43,621] {processor.py:654} WARNING - No viable dags retrieved from /opt/airflow/dags/padron.py
[2022-05-07 07:01:43,720] {processor.py:171} INFO - Processing /opt/airflow/dags/padron.py took 0.202 seconds
